src/components/SymbolSelectButton.tsx

```diff
@@ -62,12 +62,13 @@ export const SymbolInfo = observer(SymbolInfoBase);
 
 const SymbolSelectButtonBase = () => {
     const { chartTitle } = useStores();
-    const { isSymbolOpen } = chartTitle;
+    const { isSymbolOpen, currentActiveCategory } = chartTitle;
+    const is_forex_market = currentActiveCategory === 'forex';
     return (
         <div className='cq-symbol-select-btn'>
             <div className='cq-symbol-select-btn__container'>
                 <SymbolInfo />
-                {!isSymbolOpen && <div className='cq-symbol-closed-text'>{t.translate('CLOSED')}</div>}
+                {!isSymbolOpen && !is_forex_market && <div className='cq-symbol-closed-text'>{t.translate('CLOSED')}</div>}
```

Comment:
I think sometimes Forex markets can be closed from BO. Will it still show `Closed` here if it is closed in BO?

Reply:
Hmm, no, unfortunately this is required due to forex markets being marked as closed when the daily blackout happens. But in case the markets are closed from BO, the user will not be able to make any trade and it will show the error on the right side. I think the idea of the card is that the forex markets do not close in general, so if trading is stopped for some reason, it still should not be marked as closed on our side.

Reply:
@akmal-binary Looks like `Forex` market will be closed on Weekends. We still need to show `Closed` label for it.
![Screenshot 2022-03-28 at 12 01 36 PM](https://user-images.githubusercontent.com/56330681/160324548-ccc1575b-fef4-46bc-8640-b972bc9f31f0.png)

Reply:
Thanks Bala, I'll change the logic accordingly 👌🏻